### ISSUE TITLE
Can you also show images or stream audio?

### ISSUE THREAD

**Issue by Paul-Ver** [alxhotel/chromecast-api#10]
I was thinking if I can also use this library to send images to the chromecast. And can you also send sound only over HDMI (by pointing the URL to an mp3)?
It seems logical, but I didn't find it in the documentation.

I'm currently thinking of buying a chromecast and if it supports casting images, that would make the choice easier.

**Comment by alxhotel** [comment]
You are right. We should extend this library with more types of media.

I start working on this in the next week or so. But anyone is welcome to contribute.

Thanks @Paul-Ver

**Comment by 00SteinsGate00** [comment]
How about YouTube links? I tried passing in a simple YouTube link but it didn't work

**Comment by Paul-Ver** [comment]
Actually, I've been looking into this library lately.

However, I'm unable to play any other file than the example file.
For instance, I tried replacing the url with: 'http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4' but it just won't play.

I can access the file from my laptop (same network) and the demo one works. 

The log shows:
```
Playing in your chromecast
Error: connect EHOSTDOWN 192.168.55.62:8009 - Local (192.168.55.19:56324)
```

**Comment by alxhotel** [comment]
Hi @Paul-Ver, I just test it and it works correctly with your link.

This is my code:
```js
var ChromecastAPI = require('chromecast-api')
var devices = []
var browser = new ChromecastAPI.Browser()
console.log('Searching for devices')

browser.on('deviceOn', function (device) {
  if (!devices[device.host]) {
    devices[device.host] = device
    console.log('Found chromecast: `' + device.config.name + '` at ' + device.host)
  }
  device.play('http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4', 0, function () {
    console.log('Playing in chromecast: ' + device.config.name)
  })
})
```

**Comment by helloworldkr** [comment]
is there any way to cast images eg. 
 https://i.ytimg.com/vi/P0GOIAK8ces/maxresdefault.jpg

**Comment by tmigone** [comment]
If you guys are still interested several years down the road... I just implemented image support on #25

**Comment by Paul-Ver** [comment]
> If you guys are still interested several years down the road... I just implemented image support on #25

Wow.
Just Wow.

I have no write acces to this repository.
I just checked your code and it looks amazingly simple.

I've just tested with the image of @helloworldkr and it actually works!

@alxhotel please merge this or give write access to tmigone!

**Comment by Paul-Ver** [comment]
Issued solved in #25 thank you all very much for your interest!

**Comment by Atul-Anand-Jha** [comment]
can we stream youtube videos through YOUTUBE URL also ??
